javascript/src/live/helpers.js

```
@@ -1,192 +1,175 @@
+    const { startDataChunk, buffer, bit_depth, sample_rate, channels } = parseAudioFile(filePath);
+    const audioData = buffer.subarray(startDataChunk + 8, buffer.readUInt32LE(startDataChunk + 4));
+    const chunkDuration = 0.1; // 100 ms
+    const bytesPerSample = bit_depth / 8;
+    const bytesPerSecond = sample_rate * channels * bytesPerSample;
+    const chunkSize = chunkDuration * bytesPerSecond;
+    const recorder = {
+        interval: null,
+        start() {
+            let offset = 0;
+            this.interval = setInterval(() => {
+                onAudioChunk(audioData.subarray(offset, (offset += chunkSize)));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)